deploy/awstools/awstools.py

```diff
@@ -153,13 +158,53 @@ def construct_instance_market_options(instancemarket, spotinterruptionbehavior,
     else:
         assert False, "INVALID INSTANCE MARKET TYPE."
 
-def launch_instances(instancetype, count, instancemarket, spotinterruptionbehavior, spotmaxprice, blockdevices=None, tags=None, randomsubnet=False, user_data_file=None):
-    """ Launch count instances of type instancetype, optionally with additional
-    block devices mappings and instance tags
-
-         This will launch instances in avail zone 0, then once capacity runs out, zone 1, then zone 2, etc.
+def launch_instances(instancetype, count, instancemarket, spotinterruptionbehavior, spotmaxprice, blockdevices=None,
+                     tags=None, randomsubnet=False, user_data_file=None, timeout=timedelta(), always_expand=True):
+    """ Launch `count` instances of type `instancetype`
+
+    Using `instancemarket`, `spotinerruptionbehavior` and `spotmaxprice` to define instance market conditions
```

Comment:
spotinterruptionbehavior

Reply:
Thanks.  Good catch.
```suggestion
    Using `instancemarket`, `spotinterruptionbehavior` and `spotmaxprice` to define instance market conditions
```